requirements/constraints.txt

```diff
@@ -0,0 +1,351 @@
+# Requirements of requirements
+
+#
+# Required by multiple projects
+#
+
+# alembic, Sphinx
+MarkupSafe==1.0 \
+    --hash=sha256:a6be69091dac236ea9c6bc7d012beab42010fa914c459791d627dad4910eb665
+
+# alembic, boto3, factory_boy, ndg_httpsclient
+python-dateutil==2.6.1 \
+    --hash=sha256:891c38b2a02f5bb1be3e4793866c8df49c7d19baabf9c1bad62547e0b4866aca \
+    --hash=sha256:95511bae634d69bc7329ba55e646499a842bc4ec342ad54a8cdb65645a0aad3c
+
+# alembic, boto3, factory_boy, ndg_httpsclient, protobuf, Sphinx
```

Comment:
I don't know if I'd keep these comments. They're nice now because you did a lot of research, but these things change frequently and there's no good way to know when it changed and update the comment accordingly.

Reply:
OK, I'll remove them. Should ``constraints.txt`` be grouped by package, sorted by package name, or something else?

Reply:
My experience with Socorro leads me to believe that there's no good grouping that isn't hard to maintain, so I'd just let it sort itself however it likes. Hopefully there are better tools for managing things like this some day.